pkg/util/version/const.go

```diff
@@ -25,6 +25,8 @@ const (
 
 var GitCommit = "unknown"
 
+// TODO: set value for MustGather field for all streams below
```

Comment:
Ready for review, but don't merge until these stream fields have been populated.